test/unit.js

```diff
@@ -768,7 +768,7 @@ test('getProjectName with a multiple files', t => {
     nowConfig: {},
     paths: ['/tmp/aa/abc.png', '/tmp/aa/bbc.png']
   });
-  t.is(project, 'files');
+  t.is(project, 'abc.png');
```

Comment:
I'm not a huge fan of this change, mostly because I think of `now somefile.tgz` to be a temporary upload. So it makes sense that all my temporary uploads go the same project.

If I want something that should be longed lived, I can create a directory for it and then a subdir of `public` with my public files.